templates/zerver/find_my_team.html

```diff
@@ -0,0 +1,44 @@
+{% extends "zerver/portico.html" %}
+
+{% block portico_content %}
+
+<div class="app find-team-page">
+  <div class="app-main find-team-page-container">
+    <div class="find-team-form">
+      <p class="lead">
+        <h3 class="find-team-page-header">{{ _("Find your team") }}…</h3>
+        <p>
+        We'll send you an email with sign-in links for any Zulip teams
+        associated with the address(es) you enter here.
+        </p>
+      </p>
+      <form class="form-inline" id="find_my_team" name="email_form"
+            action="{{ current_url() }}" method="post">
+        {{ csrf_input }}
+        <input type="text" class="form-control required"
+               placeholder="{{ _("Enter email addresses") }}"
+               id="emails" name="emails"/>
+        <button type="submit" class="btn btn-primary">{{ _('Find team') }}</button>
+        <div><i>{{ form.emails.help_text }}</i></div>
+      </form>
+      <div id="errors"></div>
+      {% if form.emails.errors %}
+        {% for error in form.emails.errors %}
+          <div class="alert alert-error">{{ error }}</div>
+        {% endfor %}
+      {% endif %}
+      <div id="results">
+        {% if emails %}
+        <p> We have sent the login instructions to:</p>
```

Comment:
This should make clear that they won't get an email if there isn't actually an account.